arangod/Pregel/Worker/GraphStore.cpp

```diff
@@ -810,12 +819,12 @@ template class arangodb::pregel::GraphStore<double, double>;
 template class arangodb::pregel::GraphStore<float, uint8_t>;
 
 // specific algo combos
-template class arangodb::pregel::GraphStore<WCCValue, uint64_t>;
-template class arangodb::pregel::GraphStore<SCCValue, int8_t>;
+template class arangodb::pregel::GraphStore<algos::WCCValue, uint64_t>;
+template class arangodb::pregel::GraphStore<algos::SCCValue, int8_t>;
 template class arangodb::pregel::GraphStore<ECValue, int8_t>;
```

Comment:
I think it would make sense to put the code in the new files inside the Algo directory also into the namespace algos.

Reply:
Done